app/controllers/keywords_controller.rb

```diff
@@ -2,4 +2,9 @@
 
 class KeywordsController < ApplicationController
   def index; end
+
+  def create
+    @keyword = params['keyword']
+    @raw_response = GoogleService::ClientService.query(@keyword)
```

Comment:
In case of failure, I guess we should redirect to index with error.

Reply:
OKay, I gave it a try! (pushed in this branch already). 
Any feedback is of course welcome :)

Reply:
@ankitkalia1195  ready for re-review ;-)